ocaml/tests/alerts/test_alert_certificate_check.ml

```diff
@@ -26,24 +26,22 @@ let good_samples =
 
 let expiring_samples =
   [
-    ("20200302T02:00:00Z", 30, Api_messages.host_server_certificate_expiring_30)
-  ; ("20200301T02:00:00Z", 29, Api_messages.host_server_certificate_expiring_30)
-  ; ("20200215T02:00:00Z", 14, Api_messages.host_server_certificate_expiring_14)
-  ; ("20200208T02:00:00Z", 7, Api_messages.host_server_certificate_expiring_07)
-  ; ("20200201T02:00:00Z", 0, Api_messages.host_server_certificate_expiring_07)
+    ("20200302T02:00:00Z", Api_messages.host_server_certificate_expiring_30)
```

Comment:
are we sure we want to remove these numbers here?

Reply:
Ah I see, it wasn't used in the formatted message (it probably changed).

Reply:
The numbers are already encoded in the name of the message and not used to generate the message, so pretty sure ;)